cli/fmt.rs

```diff
@@ -21,21 +21,38 @@ use std::path::PathBuf;
 use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::{Arc, Mutex};
 
-fn is_supported(path: &Path) -> bool {
```

Comment:
I just moved some functions around in here so this reads top to bottom, which I think helps improve the readability of this file a little more.